source/backend/api/Services/TakeService.cs

```diff
@@ -223,7 +223,14 @@ private void RecalculateOwnership(long acquisitionFilePropertyId, PimsTake take)
             var currentTakes = _takeRepository.GetAllByPropertyId(currentProperty.PropertyId);
```

Comment:
Hmm, does this take into account expired takes? given our conversation today, we should not be incorporating expired takes into the calculation.

Reply:
Right now all the takes that could result in ownership cannot expire. Just as a reminder, the interest holder calculation is done "on-the-fly" for the views and filters and that one does take into account the expire date.

Reply:
Ah yes, did not consider that, well thats nice, as it simplifies this.